### PR TITLE
Refactor enterprise introduction and getting started

### PR DESCRIPTION
**Pull request by wivwiv**

### CODE REVIEW

File: en_US/README.md [hunk 1]
```diff
@@ -1,56 +1,146 @@
...
 
+{% endemqxce %}
+
+{% emqxee %}
+
+### 100% MQTT Compliant
+
+- 100% MQTT Compliant
+- 100% compliant with MQTT v5.0 and v3.x standards.
+- Fully supports QoS0, 1 & 2 of MQTT message delivery.
```
Comment (zmstone) [inline en_US/README.md]:
```suggestion
- Fully supports QoS 0, 1 & 2 of MQTT message delivery.
```

File: en_US/README.md [hunk 2]
```diff
@@ -1,56 +1,146 @@
...
+- Works with all MQTT clients and libraries like Eclipse Paho.
+
+### Data Security & Privacy
+
+- Data Security & Privacy
```
Comment (zmstone) [inline en_US/README.md]:
```suggestion
```

File: en_US/README.md [hunk 3]
```diff
@@ -1,56 +1,146 @@
...
+- Ensure data security with MQTT over TLS/SSL or QUIC.
+- Authenticate with LDAP, JWT, PSK, and X.509 certificates.
```
Comment (zmstone) [inline en_US/README.md]:
```suggestion
- Authenticate with LDAP, JWT, PSK, X.509 certificates, and more.
```

File: en_US/README.md [hunk 4]
```diff
@@ -1,56 +1,146 @@
...
+- Data privacy protection is in line with the EU’s GDPR.
```
Comment (zmstone) [inline en_US/README.md]:
```suggestion
- Rich data management APIs to help keeping your data protection policy in line with EU’s GDPR.
```

File: en_US/README.md [hunk 5]
```diff
@@ -1,56 +1,146 @@
...
+- Adopt cloud-native architecture based on Kubernetes. One-click deployment for on-premises or fully managed service.
+
+### Run Anywhere
+
+- Run Anywhere
+- Run anywhere in private, hybrid, and public clouds like AWS, GCP, and Azure without vendor lock-in.
+
+### High ROI with Low TCO
+
+- High ROI with Low TCO
```
Comment (zmstone) [inline en_US/README.md]: This duplicates the heading
```suggestion
```

File: en_US/README.md [hunk 6]
```diff
@@ -1,56 +1,146 @@
...
+- Efficiently use network and server resources.
+- Pay by annual subscription, no costly one-off purchases.
```
Comment (zmstone) [inline en_US/README.md]:
```suggestion
- Pay for annual subscription, no costly one-off purchases.
```

File: en_US/README.md [hunk 7]
```diff
@@ -1,56 +1,146 @@
...
+
+- Cloud-Native
+- Adopt cloud-native architecture based on Kubernetes. One-click deployment for on-premises or fully managed service.
+
+### Run Anywhere
+
+- Run Anywhere
+- Run anywhere in private, hybrid, and public clouds like AWS, GCP, and Azure without vendor lock-in.
+
+### High ROI with Low TCO
```
Comment (zmstone) [inline en_US/README.md]:
```suggestion
### High ROI (Return on Investment) with Low TCO (Total Cost of Ownership)
```

File: en_US/README.md [hunk 8]
```diff
@@ -1,56 +1,146 @@
...
+
+- High ROI with Low TCO
+- Efficiently use network and server resources.
+- Pay by annual subscription, no costly one-off purchases.
+- Get high ROI from your IoT solutions with low TCO.
+
+### Global Technical Support
+
+- Global Technical Support
+- 7 branches in USA, Europe, Japan and China.
```
Comment (zmstone) [inline en_US/README.md]:
```suggestion
- 7 sites globally, covering USA, Europe, Japan and China.
```

File: en_US/README.md [hunk 9]
```diff
@@ -1,56 +1,146 @@
...
+- More than ten partners in Europe, USA and India.
```
Comment (zmstone) [inline en_US/README.md]:
```suggestion
- More than ten business partners in Europe, USA and India.
```

File: en_US/README.md [hunk 10]
```diff
@@ -1,56 +1,146 @@
...
+### High ROI with Low TCO
+
+- High ROI with Low TCO
+- Efficiently use network and server resources.
+- Pay by annual subscription, no costly one-off purchases.
+- Get high ROI from your IoT solutions with low TCO.
+
+### Global Technical Support
+
+- Global Technical Support
```
Comment (zmstone) [inline en_US/README.md]:
```suggestion
```

File: en_US/README.md [hunk 11]
```diff
@@ -1,56 +1,146 @@
...
+### Data Security & Privacy
+
+- Data Security & Privacy
+- Ensure data security with MQTT over TLS/SSL or QUIC.
+- Authenticate with LDAP, JWT, PSK, and X.509 certificates.
+- Data privacy protection is in line with the EU’s GDPR.
+
+### Cloud-Native
+
+- Cloud-Native
```
Comment (zmstone) [inline en_US/README.md]:
```suggestion
```

File: en_US/README.md [hunk 12]
```diff
@@ -1,56 +1,146 @@
...
 - **High Availability**: Achieve high availability and horizontal scalability through a masterless distributed architecture.
 - **Cloud-Native & K8s**: Easy to deploy on-premises or in public clouds with Kubernetes Operator and Terraform.
 
+{% endemqxce %}
+
+{% emqxee %}
+
+### 100% MQTT Compliant
+
+- 100% MQTT Compliant
```
Comment (zmstone) [inline en_US/README.md]:
```suggestion
```

File: en_US/README.md [hunk 13]
```diff
@@ -1,56 +1,146 @@
...
+- Gateways
+  - CoAP
+  - LwM2M
+  - MQTT-SN
+  - Stomp
+  - GB/T 32960 (Enterprise edition) <!--cannot use 'emqxee' macro inside list-->
+  - JT/T 808 (Enterprise edition)
+  - OCPP (Enterprise edition)
+
+Add more features to MQTT:
```
Comment (zmstone) [inline en_US/README.md]:
```suggestion
Advanced features for MQTT:
```

File: en_US/README.md [hunk 14]
```diff
@@ -1,56 +1,146 @@
...
+- PSK authentication
+- Access control (Authorization) based on Client ID, IP address, and username supported using built-in database, Redis, MySQL, PostgreSQL, MongoDB as data sources, and HTTP Server to provide authorization services
+- API to ban clients
+
+### Scalability
+
 - Multi-server node cluster (Cluster)
-- Support manual, mcast, dns, etcd, k8s and other cluster discovery methods
-- Automatic network partition healing
+- Support `manual`, `mcast`, `dns`, `etcd`, `k8s` cluster discovery methods
```
Comment (zmstone) [inline en_US/README.md]: We are considering deprecating mcast.
```suggestion
- Support `manual`, `dns`, `etcd`, `k8s` cluster discovery methods
```

File: en_US/README.md [hunk 15]
```diff
@@ -1,56 +1,146 @@
...
+- Multi-server node bridge (Bridge)
+
+### Data Integration
+
+- SQL syntax data integrations to extract, filter, enrich, and transform MQTT message or internal events to formats expected by users and send to external data platforms
```
Comment (zmstone) [inline en_US/README.md]:
```suggestion
- SQL syntax data integrations to extract, filter, enrich, and transform MQTT message or internal events to desired format and export them to external data platforms
```

File: en_US/README.md [hunk 16]
```diff
@@ -1,56 +1,146 @@
...
+- Supports data bridging with other brokers or IoT platforms using MQTT(such as EMQX Cloud, AWS IoT Core, Azure IoT Hub)
```
Comment (zmstone) [inline en_US/README.md]:
```suggestion
- Supports data bridging with other brokers or IoT platforms using MQTT (such as EMQX Cloud, AWS IoT Core, Azure IoT Hub)
```

File: en_US/README.md [hunk 17]
```diff
@@ -1,56 +1,146 @@
...
+- Supports integration with other apps using WebHook
+- 30+ enterprise system integrations include Kafka, RDS, various SQL / NoSQL / time-series databases, and enterprise systems such as Oracle and SAP (Enterprise edition)
```
Comment (zmstone) [inline en_US/README.md]:
```suggestion
- 30+ enterprise system integrations including Kafka, RDS, various SQL / NoSQL / time-series databases, and enterprise systems such as Oracle and SAP (Enterprise edition)
```

File: en_US/README.md [hunk 18]
```diff
@@ -1,56 +1,146 @@
...
-  - Bridge data to MQTT Broker
-  - Check (debug)
-  - Send data to web service
+
+### Observability
+
+- Client online status query
+- Cluster status and metrics query
+- Integration with Prometheus/StatsD
+- Automatic network partition healing
```
Comment (zmstone) [inline en_US/README.md]: This is reliability
```suggestion
```

File: en_US/README.md [hunk 19]
```diff
@@ -1,56 +1,146 @@
...
+- Online log based tracing
+- Erlang runtime tracing tools
```
Comment (zmstone) [inline en_US/README.md]:
```suggestion
- Runtime tracing tools
```

File: en_US/getting-started/getting-started.md [hunk 20]
```diff
@@ -2,20 +2,37 @@
...
 which runs EMQX Enterprise.
 You can customize the deployment to suit your business development plans the best and get started quickly.
 
+{% endemqxce %}
+
+{% emqxee %}
+
+## Deployment Mode
+
+EMQX as a software provides the self-managed version [EMQX Enterprise](https://www.emqx.com/en/try?product=enterprise).
```
Comment (zmstone) [inline en_US/getting-started/getting-started.md]:
```suggestion
EMQX Enterprise as a software provides a free-of-charge non-expiry trail license. You are welcome to [download it](https://www.emqx.com/en/try?product=enterprise).
```

File: en_US/getting-started/getting-started.md [hunk 21]
```diff
@@ -2,20 +2,37 @@
 
 Starting from download and installation, this document is to guide you to quickly get a EMQX node up and running.
 
+{% emqxce %}
+
 ## Select EMQX Edition
```
Comment (zmstone) [inline en_US/getting-started/getting-started.md]:
```suggestion
## Choose EMQX Software Edition or Service
```

File: en_US/getting-started/getting-started.md [hunk 22]
```diff
@@ -2,20 +2,37 @@
...
 
 EMQ as a company offers a fully managed MQTT cloud service [EMQX Cloud](https://www.emqx.com/en/try?product=cloud)
 which runs EMQX Enterprise.
 You can customize the deployment to suit your business development plans the best and get started quickly.
 
+{% endemqxce %}
+
+{% emqxee %}
+
+## Deployment Mode
```
Comment (zmstone) [inline en_US/getting-started/getting-started.md]:
```suggestion
## Choose Your Hosting Mode
```

File: en_US/getting-started/getting-started.md [hunk 23]
```diff
@@ -115,28 +158,49 @@ For more information about Terraform installation and deployment, please refer t
...
+
+{% emqxee %}
+
+For other platforms which are not in the prebuilt list,
+you may simply [contact EMQ](https://www.emqx.com/en/contact) for support.
+{% endemqxee %}
+
+For other platforms which are not in the prebuilt list,
+you may {% emqxce %}[build the runnables from source code](./install.md#source-code-compilation-and-installation) or{% endemqxce %}
 simply [contact EMQ](https://www.emqx.com/en/contact) for support.
```
Comment (zmstone) [inline en_US/getting-started/getting-started.md]:
```suggestion
{% emqxce %}
For other platforms which are not in the prebuilt list,
you may [build the runnables from source code](./install.md#source-code-compilation-and-installation) or
simply [contact EMQ](https://www.emqx.com/en/contact) for support.
{% endemqxce %}
```

File: zh_CN/README.md [hunk 24]
```diff
@@ -15,56 +15,147 @@ ref:
...
 - **高可用**：采用 Masterless 的大规模分布式集群架构，实现系统高可用和水平扩展。
+{% endemqxce %}
+
+{% emqxee %}
+
+### 完整的 MQTT 协议
+
+- 完整的 MQTT 协议
+- 完整支持 MQTT 3.x 和 MQTT 5.0 协议标准
+- 完全支持 QoS0, 1 & 2 的 MQTT 消息传递
```
Comment (wivwiv) [inline zh_CN/README.md]:
```suggestion
- 完全支持 QoS 0, 1 & 2 的 MQTT 消息传递
```

File: zh_CN/README.md [hunk 25]
```diff
@@ -15,56 +15,147 @@ ref:
...
-  - 发送数据到 Web 服务
+
+### 可观测性
+
+- 客户端在线状态查询
+- 集群状态与指标查询
+- Prometheus 集成
+- 自动网络分区恢复
+- 在线日志追踪(Log Trace)
+- Erlang 运行时追踪工具
```
Comment (wivwiv) [inline zh_CN/README.md]:
```suggestion
- 运行时追踪工具
```

File: zh_CN/README.md [hunk 26]
```diff
@@ -15,56 +15,147 @@ ref:
...
+- 完整支持 MQTT 3.x 和 MQTT 5.0 协议标准
+- 完全支持 QoS 0, 1 & 2 的 MQTT 消息传递
+- 支持所有 MQTT 客户端和库（如 Eclipse Paho）
+
+### 数据安全与隐私
+
+- 数据安全与隐私
+- 基于 MQTT over TLS/SSL 和 QUIC 确保数据安全
+- LDAP、JWT、PSK 和 X.509 证书等多种身份认证
+- 数据隐私保护符合欧盟的 GDPR 法案
```
Comment (wivwiv) [inline zh_CN/README.md]:
```suggestion
- 丰富的管理 API 确保数据保护策略符合欧盟 GDPR 法案
```

File: zh_CN/README.md [hunk 27]
```diff
@@ -15,56 +15,147 @@ ref:
...
+- 基于 JWT 的身份认证与权限控制，支持 JWKs
+- MQTT 5.0 增强认证
+- PSK 身份验证
+- 基于 Client ID、IP 地址，用户名的访问控制，支持使用内置数据库、Redis、MySQL、PostgreSQL、MongoDB 作为数据源，也支持使用 HTTP Server 提供授权服务
+- 客户端黑名单支持
+
+### 可伸缩性
+
+- 多节点集群 (Cluster)
+- 支持手动、mcast、dns、etcd、k8s 集群发现方式集群
```
Comment (wivwiv) [inline zh_CN/README.md]:
```suggestion
- 支持手动、dns、etcd、k8s 集群发现方式集群
```

**Review by zmstone — APPROVED** [review]

**Pull request merged**